lib/split/dashboard/public/style.css

```diff
@@ -312,7 +314,7 @@ a.button.green:focus, button.green:focus, input[type="submit"].green:focus {
   background:#768E7A;
 }
 
-#filter, #clear-filter {
+#filter, #clear-filter, #toggle-completed, #toggle-active, #register-experiment-btn, #experiment-select{
```

Comment:
I know this is gross lol... i'd refactor this to a shared css class to apply the padding rather than specify every id...